src/IECoreMaya/LiveScene.cpp

```diff
@@ -70,17 +70,71 @@
 #include "maya/MTransformationMatrix.h"
 #include "maya/MDagPathArray.h"
 #include "maya/MFnNurbsCurve.h"
+#include "maya/MGlobal.h"
 
 #include "OpenEXR/ImathBoxAlgo.h"
 
 #include <boost/algorithm/string.hpp>
 #include <boost/tokenizer.hpp>
+#include <boost/format.hpp>
+
+#include <maya/MFnSet.h>
 
 using namespace IECore;
 using namespace IECoreMaya;
 
 IE_CORE_DEFINERUNTIMETYPED( LiveScene );
 
+namespace
+{
+
+void readExportableSets( std::set<SceneInterface::Name> &exportableSets, const MDagPath &dagPath )
+{
+	// convert maya sets to tags
+	MSelectionList selectionList;
+	selectionList.add( dagPath );
+
+	MObjectArray sets;
+	if( !MGlobal::getAssociatedSets( selectionList, sets ) )
```

Comment:
Does this differentiate between "object is in the set" and "one vertex is in the set"? Not sure if we need to care, but maybe we should only consider sets that contain the whole object?